src/Avalonia.Controls/AutoCompleteBox/AutoCompleteBox.Properties.cs

```diff
@@ -167,6 +167,18 @@ public partial class AutoCompleteBox
             AvaloniaProperty.Register<AutoCompleteBox, Func<string?, CancellationToken, Task<IEnumerable<object>>>?>(
                 nameof(AsyncPopulator));
 
+        /// <summary>
+        /// Defines the <see cref="InnerLeftContent"/> property
+        /// </summary>
+        public static readonly StyledProperty<object> InnerLeftContentProperty =
```

Comment:
CI is failing, because nullable attributes are wrong here. It should be "object?"